src/shell/components/global-sidebar/GlobalSidebar.tsx

```diff
@@ -296,10 +298,12 @@ const GlobalSidebar: FC<GlobalSidebarProps> = ({ onClick, openNav }) => {
           </Box>
         </Box>
       </ThemeProvider>
-      {showFaviconModal && (
+      {ui.isUpdateFaviconModalOpen && (
```

Comment:
I thought that was a better way to do it instead of passing the toggle function through 3 components just to toggle that state. The button that initially opens the favicon update modal is 3 child components from the GlobalSidebar.